{
    "ldap_attribute_already_exists": "O atributo '{attribute}' xa existe e ten o valor '{value}'",
    "invalid_usage": "Uso non válido, pass --help para ver a axuda",
    "invalid_token": "Token non válido - por favor autentícate",
    "invalid_password": "Contrasinal non válido",
    "invalid_argument": "Argumento non válido '{argument}': {error}",
    "instance_already_running": "Hai unha operación de YunoHost en execución. Por favor agarda a que remate antes de realizar unha nova.",
    "info": "Info:",
    "folder_not_exist": "O cartafol non existe",
    "folder_exists": "Xa existe o cartafol: '{path}'",
    "file_not_exist": "Non existe o ficheiro: '{path}'",
    "file_exists": "Xa existe o ficheiro: '{path}'",
    "error": "Erro:",
    "deprecated_command_alias": "'{prog} {old}' xa non se utiliza e será eliminado no futuro, usa '{prog} {new}' no seu lugar",
    "deprecated_command": "'{prog} {command}' xa non se utiliza e xa non se usará no futuro",
    "confirm": "Confirma {prompt}",
    "colon": "{}: ",
    "authentication_required_long": "Requírese autenticación para realizar esta acción",
    "authentication_required": "Autenticación requerida",
    "argument_required": "O argumento '{argument}' é requerido",
    "logged_out": "Sesión pechada",
    "password": "Contrasinal",
    "warning": "Aviso:",
    "values_mismatch": "Non concordan os valores",
    "unknown_user": "Usuaria '{user}' descoñecida",
    "unknown_group": "Grupo '{group}' descoñecido",
    "session_expired": "A sesión caducou. Volve a conectar por favor.",
    "unable_retrieve_session": "Non se puido obter a sesión porque '{exception}'",
    "unable_authenticate": "Non se puido autenticar",
    "success": "Ben feito!",
    "server_already_running": "Xa hai un servidor a funcionar nese porto",
    "root_required": "Tes que ser root para facer esta acción",
    "pattern_not_match": "Non concorda co patrón",
    "operation_interrupted": "Interrumpeuse a operación",
    "not_logged_in": "Non estás conectada",
    "logged_in": "Conectada",
    "ldap_server_down": "Non se puido acadar o servidor LDAP"
}
